doc/api/url.md

```diff
@@ -235,6 +235,19 @@ url.resolve('http://example.com/', '/one')    // 'http://example.com/one'
 url.resolve('http://example.com/one', '/two') // 'http://example.com/two'
 ```
 
+Note that `url.resolve()` method depends on the protocol given in the `from` parameter. 
+Some protocols like `https`, `file`, `ftp`, `wss` and `gopher` have special cases.
+
+For example:
+
+```js
+url.resolve('https://foo.tld', 'bar')   // 'https://foo.tld/bar'
+url.resolve('wss://foo.tld', 'bar')      // 'wss://bar'
```

Comment:
While you are at it also add semicolons mostly for consistency with other code example in the doc.